apolloschurchapp/fastlane/Fastfile

```diff
@@ -45,11 +37,9 @@ platform :ios do
     # get project name
     proj = Dir.glob("../ios/*.xcworkspace")[0][7..-13]
 
-    identifier = product_bundle_id(path: "../ios/" << proj << ".xcodeproj")
-
     app_store_connect_api_key(key_filepath: "ios/apollos.p8", in_house: false)
     setup_ci
-    match(type: 'appstore', app_identifier: [identifier, "#{identifier}.OneSignalNotificationServiceExtension"], readonly: true)
```

Comment:
you'll have to add the one signal extension bundle ID to the match app identifier env variable now. it's comma separated, no space

Reply:
So this shouldn't have been merged? Does this change look right?

Reply:
That change is correct, we just specify that variable as an env now

Reply:
Oh gotcha